src/ls/clients/regal.ts

```diff
@@ -370,13 +370,32 @@ function handleRegalShowEvalResult(params: ShowEvalResultParams) {
     Object.keys(params.result.printOutput).map(Number).forEach((line) => {
         const lineLength = activeEditor.document.lineAt(line).text.length
 
+        const printText = " 🖨️ => " + params.result.printOutput[line].join(" => ")
+
+        const joinedLines = params.result.printOutput[line].join("\n")
+
+        // pre block formatting fails if there are over 100k chars
+        let hoverText = joinedLines;
+        if (joinedLines.length < 100000) {
+            hoverText = makeCode("text", joinedLines)
+        }
+
+        const hoverMessage = "### Print Output\n\n" + hoverText
+
+        let attachmentMessage = printText
```

Comment:
The `printText` var seems unnecessary, and could be replaced here with

```typescript
let attachmentMessage  = " 🖨️ => " + params.result.printOutput[line].join(" => ")
```

Reply:
Ahh yeah, will fix.